shell/platform/fuchsia/BUILD.gn

```diff
@@ -4,25 +4,75 @@
 
 import("//build/fuchsia/sdk.gni")
 import("$flutter_root/common/config.gni")
+import("$flutter_root/tools/fuchsia/dart.gni")
+import("$flutter_root/tools/fuchsia/fuchsia_host_bundle.gni")
 
 if (using_fuchsia_sdk) {
   testonly = true
 
   product_suffix = ""
+  is_product = false
 
   if (flutter_runtime_mode == "release") {
     product_suffix = "product_"
+    is_product = true
+  }
+
+  fuchsia_host_bundle("flutter_binaries") {
+    name = "flutter-binaries"
```

Comment:
I was thinking of unifying all fuchsia related artifacts to using `-` since that seems to be the convention. That said, we could still have `flutter_binaries-linux-x64`. Btw, `flutter-binaries-linux-x64` seems better than `flutter_binaries-linux-x64`, as we don't mix underscores and hyphens, no?